lua/conform/runner.lua

```diff
@@ -152,7 +152,9 @@ end
 ---@param new_lines string[]
 ---@param range? conform.Range
 ---@param only_apply_range boolean
-M.apply_format = function(bufnr, original_lines, new_lines, range, only_apply_range)
+---@param dry_run boolean
+---@return boolean?
```

Comment:
Let's change the type to be non-nilable and add a name to the return value like so:
```lua
---@return boolean any_changes
```